src/ngraph/function.cpp

```diff
@@ -224,3 +224,19 @@ void Function::set_placement(size_t placement)
 {
     m_placement = placement;
 }
+
+// TODO(pthoreho) this will be expensive, since we will be traversing all the nodes in
+// the graph, figure out if their is a way to cache the result and invalidate/update
+// the result if the function is modified
+bool Function::is_dynamic()
```

Comment:
Could you please make this method `const`, i.e. `bool Function::is_dynamic() const`